packages/zosuss/src/SshBaseHandler.ts

```diff
@@ -41,6 +41,7 @@ export abstract class SshBaseHandler implements ICommandHandler {
 
     /**
      * Loaded z/OS SSH profile if needed
+     * @deprecated
      */
     protected mSshProfile: IProfile;
```

Comment:
FYI, compared to the previous comment, this one seems to be gone form the next branch already 😋